common/models/user.js

```diff
@@ -596,14 +607,18 @@ module.exports = function(User) {
   };
 
   User.validatePassword = function(plain) {
-    if (typeof plain === 'string' && plain) {
+    if (typeof plain === 'string' && plain && plain.length <= MAX_PASSWORD_LENGTH) {
       return true;
     }
+    if (plain.length > MAX_PASSWORD_LENGTH) {
+      err = new Error (g.f('Password too long - maximum length is %s', MAX_PASSWORD_LENGTH));
```

Comment:
Don't leak this information. Since the system does not allow passwords to be longer than 72 characters, any password longer than 72 characters is invalid.